# HG changeset patch
# User Julien Bect <jbect@users.sourceforge.net>
# Date 1508332708 -7200
#      Wed Oct 18 15:18:28 2017 +0200
# Node ID 176e5f761039901a5aa9c8504426f5af30780ad8
# Parent  4e93c39b4b97e81dc440bb4f4cc2c64dc4b876de
pt-loop.cc: Add missing header pt-jit.h

* libinterp/parse-tree/pt-loop.cc: Add missing header pt-jit.h, which is needed
  for the definition of jit_info.

diff --git a/libinterp/parse-tree/pt-loop.h b/libinterp/parse-tree/pt-loop.h
--- a/libinterp/parse-tree/pt-loop.h
+++ b/libinterp/parse-tree/pt-loop.h
@@ -25,16 +25,17 @@ along with Octave; see the file COPYING.
 
 #include "octave-config.h"
 
 class octave_value;
 
 #include "pt-cmd.h"
 #include "pt-walk.h"
 #include "symtab.h"
+#include "pt-jit.h"
 
 class jit_info;
 
 namespace octave
 {
   class tree_argument_list;
   class tree_expression;
   class tree_statement_list;
